File: mbrs/src/main/resources/static/js/services/StanicaService.js
//Fri May 11 23:51:24 CEST 2018
//Generisano na osnovu sablona: jsservice.ftl
//Element modela: GremoNaElektriko::Stanica

angular.module('gremoNaElektrikoApp.StanicaService', [])
	.factory('StanicaService', function($http) {
		var factory = {};
		
		factory.findAll = function() {
			return $http.get('/gremoNaElektriko/stanica');
		}
		
		factory.createStanica = function(stanica , gradId) {
			return $http.post('/gremoNaElektriko/stanica'+'/'+gradId, stanica);
		}

		factory.deleteStanica = function(id) {
			return $http.delete('/gremoNaElektriko/stanica/'+id);
		}
		
		factory.updateStanica = function(stanica) {
			return $http.put('/gremoNaElektriko/stanica', stanica);
		}
		
		return factory;
	});

File: mbrs/src/main/java/com/ftn/mbrs/service/StanicaService.java
//Sat May 12 11:32:37 CEST 2018
//Generisano na osnovu sablona: service.ftl
//Element modela: GremoNaElektriko::Stanica


package com.ftn.mbrs.service;

import java.util.List;

import com.ftn.mbrs.model.Stanica;

public interface StanicaService {

	public Stanica save(Stanica stanica, Long gradId);
	
	public Stanica update(Stanica stanica);

	public String delete(Long id);
	
	public Stanica findOne(Long id);
	
	public List<Stanica> findAll();
}
